core/rlp_test.go

```diff
@@ -195,3 +200,42 @@ func BenchmarkHashing(b *testing.B) {
 		b.Fatalf("hash wrong, got %x exp %x", got, exp)
 	}
 }
+
+func TestBlockRlpEncodeDecode(t *testing.T) {
+	zeroTime := uint64(0)
+
```

Comment:
How does this compare to what is in https://github.com/ethereum/go-ethereum/blob/aa636921291744af082cefbe4dd811fb58fbde63/core/types/rlp_fuzzer_test.go#L118?

Reply:
I'm specifically testing the correctness of rlp encode/decode of a block generated after isthmus hard fork is active. I'm guessing the fuzzer test would probably cover this as well? (all possible values for the withdrawals and withdrawalsHash fields?)

Reply:
I think we can move forward with merging this in the underlying branch, so we have one target of testing / rebasing. Marking this as resolved, but will give all the tests another review once into the base PR.